adapters/amx/amx.go

```diff
@@ -147,8 +147,10 @@ func (adapter *AMXAdapter) MakeBids(request *openrtb.BidRequest, externalRequest
 
 	bidResponse := adapters.NewBidderResponseWithBidsCapacity(5)
 
-	for _, sb := range bidResp.SeatBid {
-		for _, bid := range sb.Bid {
+	for _, seatbid := range bidResp.SeatBid {
+		sb := seatbid
+		for _, bidExternal := range sb.Bid {
+			bid := bidExternal
 			bidExt, bidExtErr := getBidExt(bid.Ext)
```

Comment:
Nice catch. We'll need to keep a better eye out for these issues in the future.

The `sb := seatbid` local assignment isn't necessary since you're not taking the address of it.

A common pattern for the local assignment is to use the same variable name and rely on shadowing. You may leave the names as you wish, but just wanted to throw it out there:

```
for _, bid := range sb.Bid {
  bid := bid
```

Reply:
Thanks -- that makes sense. Just pushed a commit removing that sb := assignment & changing to bid := bid